Mathlib/Topology/Instances/ENNReal/Lemmas.lean

```diff
@@ -579,6 +579,8 @@ protected theorem hasSum : HasSum f (⨆ s : Finset α, ∑ a ∈ s, f a) :=
 protected theorem summable : Summable f :=
   ⟨_, ENNReal.hasSum⟩
 
+macro_rules | `(tactic| gcongr_discharger) => `(tactic| apply ENNReal.summable)
```

Comment:
I'm not sure if it's the right way to deal with this.

Reply:
Let me think about this: the remainder of this PR is clearly good (and I'd maintainer merge that immediately).

Reply:
Medium-term, imho the correct fix is to make the discharger for `gcongr` configurable (like e.g. `fun_prop` does). In the short-term, this seems fine.

Reply:
One detail I'm wondering: will this addition be global (i.e., persist into downstream files)? That would seem unfortunate.

Reply:
Yes, it is, otherwise `gcongr` applies `Summable.tsum_le_tsum` and leaves `Summable` goals around. There are no priorities for `gcongr` lemmas, so we can't make it prefer `ENNReal.tsum_le_tsum` but still use `Summable.tsum_le_tsum` otherwise..